app/controllers/users_controller.rb

```diff
@@ -0,0 +1,6 @@
+class UsersController < ApplicationController
+    
+    def show
+      render json: current_user.as_json(:include => [{:games => {:include => [:sessions]}}, {:sessions => {:include => [:user]}}, {:shared_sessions => {:include => [:user]}}]) || false
+    end
+end
```

Comment:
Nitpicky, but it looks like you are missing an empty line at the end of this file.